### PR TITLE
Update for system init and header file

### PR DESCRIPTION
**Pull request by Fivvee0**
Some changes to vga.c vga.h, system init and linker

### CODE REVIEW

**Comment by gorciu-official** [comment]
hi

**Comment by gorciu-official** [comment]
why are you commiting this?

**Comment by Fivvee0** [comment]
because this code is easier to read

File: drivers/vga.h [hunk 1]
```diff
@@ -7,6 +7,11 @@
 #define VGA_HEIGHT 25
 #define VGA_ADDRESS 0xB8000
 
+static uint16_t* vga_buffer = (uint16_t*)VGA_ADDRESS;
+static uint8_t vga_color = VGA_COLOR_LIGHT_GREY | (VGA_COLOR_BLACK << 4);
+static uint16_t vga_row = 0;
+static uint16_t vga_column = 0;
+
```
Comment (gorciu-official) [inline drivers/vga.h:14]: i don't see any change

File: init/start.c [hunk 2]
```diff
@@ -11,7 +11,7 @@
 /**
  * Main function. Here kernel will start its work. 
  */
-void main() {
+void init_system() {
```
Comment (gorciu-official) [inline init/start.c:14]: why are you changing it? it is okay

**Review by gorciu-official — CHANGES_REQUESTED** [review]
i don't see any sense change, requesting changes for now

**Comment by Fivvee0** [comment]
but with this changes code is easier to read and you did accepted very small changes with gitignore and better issue template config

**Comment by gorciu-official** [comment]
so okay

**Pull request merged**